build.sbt

```diff
@@ -1,18 +1,18 @@
 import sbtassembly.{AssemblyUtils, MergeStrategy}
 
-val linkerdVersion = "1.5.1"
+val linkerdVersion = "1.6.1"
 
 def twitterUtil(mod: String) =
-  "com.twitter" %% s"util-$mod" %  "18.9.1"
+  "com.twitter" %% s"util-$mod" %  "18.12.0"
 
 def finagle(mod: String) =
-  "com.twitter" %% s"finagle-$mod" % "18.9.1"
+  "com.twitter" %% s"finagle-$mod" % "18.12.0"
 
 def telemetry(mod: String) =
   "io.buoyant" %% s"telemetry-$mod" % linkerdVersion
 
 def zipkin(mod: String) =
-  "io.zipkin.finagle2" %% s"zipkin-finagle-$mod" % "2.0.5"
+  "io.zipkin.finagle2" %% s"zipkin-finagle-$mod" % "2.0.8"
```

Comment:
2.0.11 was actually the current rev, fyi

Reply:
whoops nevermind.. the finagle versions are latest here!

Reply:
Yea, I wanted to use the latest version but decided that it is probably best to make sure all the finagle version aligns with finagle-zipkin's version.